frontend/server/src/test/java/org/pytorch/serve/SnapshotTest.java

```diff
@@ -368,11 +368,11 @@ private void updateSnapshot(Properties prop) {
         Snapshot snapshot = GSON.fromJson(prop.getProperty("model_snapshot"), Snapshot.class);
         snapshot.setName("snapshot");
         snapshot.setCreated(123456);
-        for (Map.Entry<String, Map<String, ModelSnapshot>> modelMap :
+        for (Map.Entry<String, Map<String, JsonObject>> modelMap :
                 snapshot.getModels().entrySet()) {
-            for (Map.Entry<String, ModelSnapshot> versionModel : modelMap.getValue().entrySet()) {
-                versionModel.getValue().setMinWorkers(4);
-                versionModel.getValue().setMaxWorkers(4);
+            for (Map.Entry<String, JsonObject> versionModel : modelMap.getValue().entrySet()) {
+                versionModel.getValue().addProperty("minWorkers", 4);
```

Comment:
Same here. const strings / prop names to either global location or use getter methods.